Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -2636,59 +2533,50 @@ public List<BaseItem> GetItemList(InternalItemsQuery query)
                 }
             }
 
-            var items = new List<BaseItem>();
-            using (var connection = GetConnection(true))
+            using var connection = GetConnection();
```

Comment:
```suggestion
            using var connection = GetConnection(true);
```
Please check again in case you missed more